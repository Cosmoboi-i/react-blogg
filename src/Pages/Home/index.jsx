import React from "react";
import { Main } from "../../components";
import { BlogPostProvider } from "../../contexts/BlogPosts/index";

export default function Home() {
  return (
    <BlogPostProvider>
      <Main />
    </BlogPostProvider>
  );
}
