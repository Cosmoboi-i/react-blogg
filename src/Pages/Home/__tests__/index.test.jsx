import { render } from "@testing-library/react";
import React from "react";
import Home from "..";

describe("Error Page", () => {
  it("should render the Home page correctly", () => {
    const { asFragment } = render(<Home />);
    expect(asFragment()).toMatchSnapshot();
  });
});
