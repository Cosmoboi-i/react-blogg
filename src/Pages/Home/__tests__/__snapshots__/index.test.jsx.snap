// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Page should render the Home page correctly 1`] = `
<DocumentFragment>
  <div
    class="Loading"
  >
    Loading...
  </div>
</DocumentFragment>
`;
